youtube2zim/scraper.py

```diff
@@ -608,47 +607,97 @@ def download_from_cache(self, key, video_path):
         logger.info(f"downloaded {video_path} from cache at {key}")
         return True
 
-    def upload_to_cache(self, key, video_path):
+    def upload_to_cache(self, key, video_path, enc_version):
         """ whether it successfully uploaded to cache """
         try:
             self.s3_storage.upload_file(
-                video_path, key, meta={"encoder_version": ENCODER_VERSION}
+                video_path, key, meta={"encoder_version": enc_version}
             )
         except Exception as exc:
             logger.error(f"{key} failed to upload to cache: {exc}")
             return False
         logger.info(f"uploaded {video_path} to cache at {key}")
         return True
 
+    def post_process_video(self, video_dir, video_id, enc, skip_recompress=False):
```

Comment:
use `preset` as that's what it is and `enc` means nothing.
I also think this is a very independent block that doesn't need to be in the scraper class.
We can restore the `low_quality` and `video_format` params and move that off the class.